languages/ruby/exercises/concept/exceptions/simple_calculator.rb

```diff
@@ -0,0 +1,12 @@
+class SimpleCalculator
+  ALLOWED_OPERATIONS = ['+', '-', '/', '*']
+
+  # TODO: Show how to define custom exceptions
+  # TODO: Explain why custom errors are subclasses of StandardError
+  # TODO: Explain why is not a good idea to rescue from Exception
```

Comment:
Nice to have some TODOs defined. These will be removed once this PR is no longer a draft, right?

Reply:
Yes, I use them to keep ideas. I'll remove them when the draft is done

Reply:
Excellent!